<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-L24K6OI" failures="0" tests="6" name="tests.DropDownTest" time="0.013" errors="1" timestamp="2024-09-29T12:54:08 IST" skipped="5">
  <testcase classname="tests.DropDownTest" name="launchurl" time="0.013">
    <error message="Cannot invoke &quot;org.openqa.selenium.WebDriver.get(String)&quot; because &quot;baselibrary.Baselibrary.driver&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebDriver.get(String)" because "baselibrary.Baselibrary.driver" is null
at baselibrary.Baselibrary.geturl(Baselibrary.java:45)
at tests.DropDownTest.launchurl(DropDownTest.java:14)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- launchurl -->
  <system-out/>
  <testcase classname="tests.DropDownTest" name="close" time="0.000">
    <skipped/>
  </testcase> <!-- close -->
  <system-out/>
  <testcase classname="tests.DropDownTest" name="practice" time="0.000">
    <skipped/>
  </testcase> <!-- practice -->
  <system-out/>
  <testcase classname="tests.DropDownTest" name="widget" time="0.000">
    <skipped/>
  </testcase> <!-- widget -->
  <system-out/>
  <testcase classname="tests.DropDownTest" name="selectmenu" time="0.000">
    <skipped/>
  </testcase> <!-- selectmenu -->
  <system-out/>
  <testcase classname="tests.DropDownTest" name="slectoptionclick" time="0.000">
    <skipped/>
  </testcase> <!-- slectoptionclick -->
  <system-out/>
</testsuite> <!-- tests.DropDownTest -->
